PowerEditor/src/WinControls/Preference/preference.rc

```diff
@@ -272,28 +272,37 @@ IDD_PREFERENCE_SUB_LANGUAGE DIALOGEX 115, 10, 460, 205
-    LISTBOX         IDC_LIST_ENABLEDLANG,24,44,78,120,LBS_NOINTEGRALHEIGHT | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON      "->",IDC_BUTTON_REMOVE,109,76,25,14
-    PUSHBUTTON      "<-",IDC_BUTTON_RESTORE,109,107,25,14
-    CTEXT           "Disabled items",IDC_DISABLEDITEMS_STATIC,144,31,72,8
-    LISTBOX         IDC_LIST_DISABLEDLANG,141,44,78,120,LBS_NOINTEGRALHEIGHT | WS_VSCROLL | WS_TABSTOP
-    CONTROL         "Treat backslash as escape character for SQL",IDC_CHECK_BACKSLASHISESCAPECHARACTERFORSQL,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,14,184,217,10
-
-    GROUPBOX        "Indent Settings",IDC_TABSETTING_GB_STATIC,242,0,211,195,BS_CENTER
-    LISTBOX         IDC_LIST_TABSETTNG,307,15,84,70,LBS_NOINTEGRALHEIGHT | WS_VSCROLL | WS_TABSTOP
-    GROUPBOX        "",IDC_GR_TABVALUE_STATIC,254,92,188,92,BS_CENTER
-    CONTROL         "Use default value",IDC_CHECK_DEFAULTTABVALUE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,250,91,122,10
-    RTEXT           "Indent size:",IDC_TABSIZE_STATIC,262,105,98,8
-    EDITTEXT        IDC_EDIT_TABSIZEVAL,362,103,14,12,ES_CENTER | ES_NUMBER
-    LTEXT           "Indent using:",IDC_INDENTUSING_STATIC,259,118,135,8
-    CONTROL         "Tab character",IDC_RADIO_USINGTAB,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,264,130,165,10
-    CONTROL         "Space character(s)",IDC_RADIO_REPLACEBYSPACE,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,264,143,165,10
-    CONTROL         "Backspace key unindents instead of removing single space",IDC_CHECK_BACKSPACEUNINDENT,"Button",BS_AUTOCHECKBOX | BS_MULTILINE | WS_TABSTOP,258,158,180,21
+    GROUPBOX        "Language Menu",IDC_CHECK_LANGMENU_GR_STATIC,120,0,220,181,BS_CENTER
+    CONTROL         "Make language menu compact",IDC_CHECK_LANGMENUCOMPACT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,134,14,174,10
+    CTEXT           "Available items",IDC_ENABLEDITEMS_STATIC,138,31,72,8
+    LISTBOX         IDC_LIST_ENABLEDLANG,133,44,78,120,LBS_NOINTEGRALHEIGHT | WS_VSCROLL | WS_TABSTOP
+    PUSHBUTTON      "->",IDC_BUTTON_REMOVE,218,76,25,14
+    PUSHBUTTON      "<-",IDC_BUTTON_RESTORE,218,107,25,14
+    CTEXT           "Disabled items",IDC_DISABLEDITEMS_STATIC,253,31,72,8
+    LISTBOX         IDC_LIST_DISABLEDLANG,250,44,78,120,LBS_NOINTEGRALHEIGHT | WS_VSCROLL | WS_TABSTOP
+    CONTROL         "Treat backslash as escape character for SQL",IDC_CHECK_BACKSLASHISESCAPECHARACTERFORSQL,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,123,184,217,10
 END
 
+IDD_PREFERENCE_SUB_INDENTATION DIALOGEX 115, 10, 460, 205
+STYLE DS_SETFONT | DS_FIXEDSYS | DS_CONTROL | WS_CHILD
+FONT 8, "MS Shell Dlg", 0, 0, 0x1
+BEGIN
+    GROUPBOX        "Auto-indent",IDC_GROUPSTATIC_AUTOINDENT,45,0,124,78,BS_CENTER
+    CONTROL         "None",IDC_RADIO_AUTOINDENT_NONE,"Button",BS_AUTORADIOBUTTON | WS_GROUP,52,22,99,10
+    CONTROL         "Basic",IDC_RADIO_AUTOINDENT_BASIC,"Button",BS_AUTORADIOBUTTON,52,36,99,10
+    CONTROL         "Advance",IDC_RADIO_AUTOINDENT_ADVANCE,"Button",BS_AUTORADIOBUTTON,52,51,99,10

```

Comment:
Proper English here is `Advanced` rather than `Advance`.

Although not strictly necessary, suggest also changing `IDC_RADIO_AUTOINDENT_ADVANCE` to `IDC_RADIO_AUTOINDENT_ADVANCED`